src/frontend/components/SearchBar/SearchBar.css

```diff
@@ -1,24 +1,45 @@
-.SearchBar {
-    display: flex;
-    flex-flow: column;
+.search-bar :focus {
+  width: 99%;
+  border: 1px solid #195280;
+  box-sizing: border-box;
 }
 
 .search-bar {
-    width: 70%;
-    height: 30px;
-    border: black 1px solid;
-    margin: 5px auto;
-    
-
+  width: 100%;
+  height: 70px;
+  margin: 1.5px;
+  background-color: #383838;
+  padding-left: 20px;
+  font-size: 20px;
+  color: #cccccc;
```

Comment:
We can use the shorthand `color: #ccc;` here

Reply:
It changes automatically